projects/angular/src/data/datagrid/_datagrid.clarity.scss

```
@@ -1942,67 +1147,32 @@
       content: '';
       display: inline-block;
       position: absolute;
-      border: variables.$clr_baselineRem_10px solid transparent;
-      @include mixins.css-var(
-        border-right-color,
-        clr-table-border-color,
-        tables-variables.$clr-table-border-color,
-        variables.$clr-use-custom-properties
-      );
-      top: variables.$clr_baselineRem_0_5 - variables.$clr_baselineRem_2px;
+      border: tokens.$cds-global-space-5 solid transparent;
+      border-right-color: tables-variables.$clr-table-border-color;
+      top: tokens.$cds-global-space-6 - tokens.$cds-global-space-2;
```

You may ignore my suggestion, but looks more clean with the mixin, at least to me.

```suggestion
      top: mixins.baselinePx(10);
```